api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -47,7 +47,11 @@ public String readMandrillApiKey() {
 
   @Override
   public String getMoodleApiKey() {
-    return readCredentialsBucketString("moodle-key.txt");
+    if (configProvider.get().featureFlags.enableMoodleV2Api) {
```

Comment:
names up for debate. it's easy enough to change this, but the keys are different and I do have to go push files around in GCS to change the name.